src/app/core/pipes/filterByDate.pipe.ts

```diff
@@ -0,0 +1,17 @@
+import {
+  Pipe,
+  PipeTransform
+ } from '@angular/core';
+
+ import _ from 'lodash';
+
+@Pipe({
+  name: 'myFilterByDate',
+  pure: false,
+})
+
+export class FilterByDatePipe implements PipeTransform {
```

Comment:
Ej, bo się zdenerwuje :P 
Dobrze, że zmieniłeś pipe: `myFilterByName` bo on robi filtrowanie, ale w tym pipie powinno zostać `orderByDate` bo to jest sortowanie.